data/reusables/actions/jobs/section-defining-outputs-for-jobs.md

```diff
@@ -40,3 +34,49 @@ jobs:
 ```
 
 {% endraw %}
+
+{% note %}
+
+**Note:** When using strategy matrix, job outputs will be combined from all jobs inside matrix. The last executed job outputs will override matching outputs from previous jobs within same matrix.
+
+{% endnote %}
+
+{% raw %}
+
+```yaml
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    outputs:
+      output_1: ${{ steps.gen_output.outputs.output_1 }}
+      output_2: ${{ steps.gen_output.outputs.output_2 }}
+      output_3: ${{ steps.gen_output.outputs.output_3 }}
+    strategy:
+      matrix:
+        job: [1, 2, 3]
```

Comment:
We should avoid overloading the term "job" to avoid confusion. Perhaps a practical example would be useful, maybe using strings?  I'll leave it up to you

Reply:
You mean something like this?
```
    strategy:
      matrix:
        tasks: ["generate-quote", "generate-order", "generate-email"]
```

Reply:
Something like we use in the [main matrix docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-a-matrix-for-your-jobs#using-a-matrix-strategy) would be good:

```yml
strategy:
  matrix:
    version: [10, 12, 14]
```

Reply:
Done